### ISSUE TITLE
Choose compile directory as command line argument instead of header in file.

### ISSUE THREAD

**Issue by AdoHTQ** [SethBling/cbscript#17]
It would probably be better to choose the compile directory as a command line argument instead of as a header so that you can easily open source a CBScript project without having to remove the header. This could be implemented so if a file does have a header, it would take priority over the argument path to keep backwards compatibility.

**Comment by ckohen** [comment]
It's still backwards compatible if the cli arg takes precedence. If you provide the arg, it's because you want it to go there, if you don't, try to find it in the file, otherwise error.

**Comment by AdoHTQ** [comment]
Yeah, either would work. Also, the command being longer by having 2 paths in it doesn't really matter because you only have to run the command once, even for multiple compiles.
